doc/github-workflow.md

```diff
@@ -0,0 +1,35 @@
+
+# GitHub workflow guidelines
+
+## Project Boards
+- One JValue-Scope Project board
+- One for each repository, linked to issues
+- One project board for each student project in the corresponding repo, possibly linked to issues 
+  
+## Issues
+- One issue board for each repository
+- Issues should be tagged appropriately to enable external contributions
+
+## External Contributions
+- External contributions are to be encouraged
+- External contributors may stage issues or work on existing ones
+- For integration into ods codebase they need to sign a contributers agreement (via the CLA Assistant GitHub Plugin)
+- Code integration and review is to be done by pull requests. The pull requests have to be approved by at least one member of the repositories admins team. 
+
+## Repositories
+- Master branch is protected: No direct push, only via reviewed pull request from separate branches
+- If possible, pull requests should be linked to issues
+- Students work on their own branches using regular pull requests to get fast feedback
+- Separate student forks only if the student projects makes it necessary. This enables fast integration of changes and prevents extensive integration effort
```

Comment:
Not sure if it really makes a difference for us.... They could just fork and send PRs

Reply:
In the end, this is probably a matter of taste. If you prefer the fork solution, I would suggest using that.